core/src/components/modal/gestures/swipe-to-close.ts

```diff
@@ -39,16 +39,18 @@ export const createSwipeToCloseGesture = (
   };
 
   const onMove = (detail: GestureDetail) => {
-    const step = detail.deltaY / height;
-    if (step < 0) { return; }
+    const step = getStep(detail.deltaY, height);
```

Comment:
We can just use the `clamp` function here instead:


```suggestion
    const step = clamp(0.0001, detail.deltaY / height, 0.9999);
```

This will ensure that the "ending" values are always in-bounds